templates/common/infra/bicep/core/host/appservice.bicep

```diff
@@ -33,16 +33,10 @@ param numberOfWorkers int = -1
 param scmDoBuildDuringDeployment bool = false
 param use32BitWorkerProcess bool = false
 
-var abbrs = loadJsonContent('../../abbreviations.json')
-var tags = { 'azd-env-name': environmentName }
-var resourceToken = toLower(uniqueString(subscription().id, environmentName, location))
-
-var prefix = contains(kind, 'function') ? abbrs.webSitesFunctions : abbrs.webSitesAppService
```

Comment:
I'm not finding `abbrs.webSitesFunctions` anymore in the PR.  Is it totally gone?

Reply:
It's there now, had to do piece meal.